guides/source/association_basics.md

```diff
@@ -140,210 +236,198 @@ create_table :books do |t|
 end
 ```
 
-### The `has_one` Association
+This ensures that even if `optional: true` allows `author_id` to be NULL, when
+it is not NULL, it must point to a valid record in the authors table.
 
-A [`has_one`][] association indicates that one other model has a reference to this model. That model can be fetched through this association.
+#### Methods Added by `belongs_to`
 
-For example, if each supplier in your application has only one account, you'd declare the supplier model like this:
+In database terms, the `belongs_to` association says that this model's table
+contains a column which represents a reference to another table. This can be
+used to set up one-to-one or one-to-many relations, depending on the setup. If
+the table of the other class contains the reference in a one-to-one relation,
+then you should use `has_one` instead.
 
-```ruby
-class Supplier < ApplicationRecord
-  has_one :account
-end
-```
+When you declare a `belongs_to` association, the declaring class automatically
+gains numerous methods related to the association. Some of these include:
```

Comment:
```suggestion
gains numerous methods related to the association. Some of these are:
```

Reply:
Updated this sentence for all of the association method sections, better readability -thanks!